js/links.js

```diff
@@ -1,46 +0,0 @@
-$(document).ready(function() {
-
-	 var aChildren = $("nav li").children(); // find the a children of the list items
-    var aArray = []; // create the empty aArray
```

Comment:
Not sure why you need aArray at all. I think you can get rid of this for loop and the one on line 16, and just replace the second for loop with the following (since all of the links in your menu have the same height and position from the top, there's no need to add/remove the class from them individually)

var divPos = $(".menu a").offset().top; // get the offset of the div from the top of page
var divHeight = $(".menu a").height(); // get the height of the div in question
if (windowPos >= divPos && windowPos < (divPos + divHeight)) {
               $(".menu a").addClass("under");
} else {
               $(".menu a").removeClass("under");
}